### PR TITLE
Fix Edit Web page error

### PR DESCRIPTION
**Pull request by mix5003**

### CODE REVIEW

**Comment by mix5003** [comment]
@serghey-rodin can you look at this please i think this is urgent too.

File: web/api/v1/edit/web/index.php [hunk 1]
```diff
@@ -260,7 +260,7 @@
     }
 
     // Add proxy support
-    if ((!empty($_SESSION['PROXY_SYSTEM'])) && (empty($v_proxy)) && (!empty($_POST['v_proxy'])) && (empty($_SESSION['error_msg']))) {
+    if ((!empty($_SESSION['PROXY_SYSTEM'])) && (empty($v_proxy)) && (!empty($_POST['v_proxy_template'])) && (empty($_SESSION['error_msg']))) {
```
Comment (mix5003) [inline web/api/v1/edit/web/index.php:263]: i not sure about this line but this works for me

**Comment by serghey-rodin** [comment]
@mix5003 thanks
@anton-reutov, @Ozea and I are currently checking this and most likely release a new build within a couple of hours

File: web/api/v1/edit/web/index.php (hunk 1 above)
Comment (Ozea) [reply web/api/v1/edit/web/index.php:263]: The `!empty($_POST['v_proxy'])` should be kept. This issue is coming from the client part only. We have to add Proxy Support checkbox in order to handle this value. Thus, checking for `!empty($_POST['v_proxy_template'])` is not accurate. Anyway, thanks for your help!

**Pull request merged**